assets/javascripts/discourse/templates/group/assigned.hbs

```diff
@@ -12,9 +12,23 @@
       </div>
     {{/if}}
     {{#load-more selector=".activity-nav li" action=(action "loadMore")}}
-      {{group-assigned-filter show-avatar=false filter="everyone" routeType=route_type assignmentCount=group.assignment_count}}
+      {{group-assigned-filter
+        show-avatar=false
+        filter="everyone"
+        routeType=route_type
+        assignment_count=group.assignment_count
```

Comment:
Does this work? I think we just changed this to `assignmentCount` rather than `assignment_count`

Reply:
Yaa you are right😅 I just added a commit changing this